From: Tias Guns <tias@ulyssis.org>
Date: Mon, 19 Mar 2012 23:35:34 +0000
Subject: [PATCH] fix swapon, swapoff: comment out MNTOPT_NOAUTO

patch from 'swap-on-off' by Dan Drown
"syscalls for swapon/swapoff and defines"
http://dan.drown.org/android/src/busybox/
---
 util-linux/swaponoff.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/util-linux/swaponoff.c b/util-linux/swaponoff.c
index 54867ec..03f7786 100644
--- a/util-linux/swaponoff.c
+++ b/util-linux/swaponoff.c
@@ -93,7 +93,9 @@ static int do_em_all(void)
 			/* swapon -a should ignore entries with noauto,
 			 * but swapoff -a should process them */
 			if (applet_name[5] != 'n'
+#ifdef MNTOPT_NOAUTO
 			 || hasmntopt(m, MNTOPT_NOAUTO) == NULL
+#endif
 			) {
 				err += swap_enable_disable(m->mnt_fsname);
 			}
-- 
1.7.10.4

